docs/AUTHENTICATION.md

```diff
@@ -0,0 +1,43 @@
+## Authentication
+Plio Analytics uses Auth0 authentication to manage access tokens and valid API requests from the client app, which in Plio's ecosystem is the frontend app.
+
```

Comment:
Are we using auth0.js or auth sdk spa?

Reply:
Or we are ignoring it for now as it seems to be important for custom claims? https://cube.dev/docs/security/jwt/auth0#create-an-application-custom-claims

Reply:
Yeah we don't need custom claims as of now. We're just relying on Auth0 for access token and auth verification at cubejs.